x-pack/plugins/security_solution/public/timelines/components/timeline/tabs_content/index.tsx

```diff
@@ -250,14 +262,18 @@ const ActiveTimelineTab = memo<ActiveTimelineTabProps>(
               overflow: hidden !important;
             `}
           >
-            <AssistantTab
-              renderCellValue={renderCellValue}
-              rowRenderers={rowRenderers}
-              timelineId={timelineId}
-              shouldRefocusPrompt={
-                showTimeline && activeTimelineTab === TimelineTabs.securityAssistant
-              }
-            />
+            {(activeTimelineTab === TimelineTabs.securityAssistant ||
+              hasTimelineConversationStarted) && (
```

Comment:
The `hasTimelineConversationStarted` condition prevents Timeline's instance of the assistant from being unmounted after the `Security assistant` has been focused for the first time.